docs/src/guide/installation.md

```diff
@@ -172,8 +172,8 @@ Install the [pre-built ARM toolchain](https://github.com/osx-cross/homebrew-arm)
 
 ```sh
 brew tap osx-cross/arm
-brew install arm-gcc-bin@12
-brew install openocd --HEAD
+brew install --force arm-gcc-bin@12
```

Comment:
Does that really fix the problem?

Reply:
No, it should be install then link --force like for AVR.

Reply:
fixed